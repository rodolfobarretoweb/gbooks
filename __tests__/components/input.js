import React from 'react';
import Input from '../../src/components/input';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Input type = 'search' label = 'field'/>
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});
