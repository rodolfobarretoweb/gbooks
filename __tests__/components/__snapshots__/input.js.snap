// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form-group"
>
  <label
    className="control-label"
    htmlFor={undefined}
  >
    field
  </label>
  <input
    className="form-control"
    id={undefined}
    type="search"
  />
</div>
`;
